canonical-web-and-design/juju.is
# Update the blog section

## Done
Update the blog section to match [the design](https://zpl.io/V1xDJ7k)

## QA
- Open the demo and go to /blog
- Check the page matches the design
- Check out the article page and see it looks good

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1990

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/116706242-5551ae80-a9c5-11eb-9764-8c0c8957270d.png)

![image](https://user-images.githubusercontent.com/1413534/116706289-639fca80-a9c5-11eb-9fa3-b2ca4b44099d.png)
